src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Shadow/PlatformShadowEffect.uwp.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System;
+using System.ComponentModel;
+using System.Linq;
+using System.Numerics;
+using Windows.UI.Composition;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Hosting;
+using Windows.UI.Xaml.Shapes;
+using Xamarin.CommunityToolkit.Effects;
+using Xamarin.CommunityToolkit.UWP.Effects;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.UWP;
+using Grid = Xamarin.Forms.Grid;
+using Image = Windows.UI.Xaml.Controls.Image;
+
+[assembly: ExportEffect(typeof(PlatformShadowEffect), nameof(ShadowEffect))]
+
+namespace Xamarin.CommunityToolkit.UWP.Effects
+{
+	public class PlatformShadowEffect : PlatformEffect
+	{
+		enum ShadowEffectState
+		{
+			Initialized,
+			PanelCreated,
+			Attached
+		}
+
+		const float defaultRadius = 10f;
+
+		const float defaultOpacity = 1f;
+
+		ShadowEffectState state;
+
+		SpriteVisual? spriteVisual;
+
+		Layout<View>? shadowPanel;
+
+		DropShadow? shadow;
+
+		FrameworkElement View => Control ?? Container;
+
+		protected override void OnElementPropertyChanged(PropertyChangedEventArgs args)
+		{
+			base.OnElementPropertyChanged(args);
+
+			switch (args.PropertyName)
+			{
+				case nameof(ShadowEffect.ColorPropertyName):
+				case nameof(ShadowEffect.OpacityPropertyName):
+				case nameof(ShadowEffect.RadiusPropertyName):
+				case nameof(ShadowEffect.OffsetXPropertyName):
+				case nameof(ShadowEffect.OffsetYPropertyName):
```

Comment:
I spotted an issue with that (and exists on iOS and Android as well, but I fixed it in my PR)

https://github.com/xamarin/XamarinCommunityToolkit/pull/1109/files#r599173646

Please amend it for UWP

Also, Please add nameof(VisualElement.Width) and nameof(VisualElement.Height)